pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1057,7 +1057,6 @@ var _ = Describe("Converter", func() {
 				UseEmulation:          true,
 				IsBlockPVC:            isBlockPVCMap,
 				IsBlockDV:             isBlockDVMap,
-				SRIOVDevices:          map[string][]string{},
```

Comment:
Please add a test verifying the `SRIOVDevices` from the context are passed to the domain.

Reply:
Done